### PR TITLE
Add Apple Pay button color option

### PR DESCRIPTION
**Pull request by dimitriBouteille**
## Summary

This PR adds the ability to choose the style of the Apple Pay button. In addition, the GooglePay and Apple Pay methods are separated into 2 different groups to improve configuration management.

Capture d’écran du 2023-11-02 15-56-18
Capture d’écran du 2023-11-02 15-56-28

Issue: #50

### CODE REVIEW

**Review by peterojo — APPROVED** [review]

File: Block/ApplePay/Shortcut/Button.php [hunk 1]
```diff
@@ -58,19 +58,24 @@ class Button extends AbstractButton implements ShortcutInterface
...
+     * @param \Magento\Checkout\Model\Session $checkoutSession
+     * @param \Magento\Payment\Model\MethodInterface $payment
+     * @param \Magento\Framework\UrlInterface $url
+     * @param \Magento\Customer\Model\Session $customerSession
+     * @param \Magento\Store\Model\StoreManagerInterface $storeManagerInterface
+     * @param \Magento\Checkout\Model\DefaultConfigProvider $defaultConfigProvider
+     * @param \Magento\Framework\App\Config\ScopeConfigInterface $scopeConfig
+     * @param \Adyen\Payment\Helper\Data $adyenHelper
+     * @param \Adyen\Payment\Helper\Config $adyenConfigHelper
+     * @param \Adyen\ExpressCheckout\Model\ConfigurationInterface $configuration
```
Comment (hossam-adyen) [inline Block/ApplePay/Shortcut/Button.php:78]: You can keep the parameters without the full namespace  and just `use` the new one in the on the top of the file 
```
use Adyen\ExpressCheckout\Model\ConfigurationInterface
```

```suggestion
     * @param Context $context
     * @param Session $checkoutSession
     * @param MethodInterface $payment
     * @param UrlInterface $url
     * @param CustomerSession $customerSession
     * @param StoreManagerInterface $storeManagerInterface
     * @param DefaultConfigProvider $defaultConfigProvider
     * @param ScopeConfigInterface $scopeConfig
     * @param AdyenHelper $adyenHelper
     * @param ConfigurationInterface $configuration
     * @param AdyenConfigHelper $adyenConfigHelper
     * @param ConfigurationInterface $configuration

```

**Comment by dimitriBouteille** [comment]
`config.buttonColor ?? 'black'`, must be used in `view/frontend/web/js/helpers/getApplePayStyles.js`

**Review by hossam-adyen — APPROVED** [review]
Thanks for your contribution @dimitriBouteille

**Comment by dimitriBouteille** [comment]
Thank you @hossam-adyen ,  I corrected the feedback and made the correction for this comment https://github.com/Adyen/adyen-magento2-express-checkout/pull/51#issuecomment-1798233930.

For me it's good :)

**Review by hossam-adyen — APPROVED** [review]

**Review by khushboo-singhvi — APPROVED** [review]

**Pull request merged**